tests/utils.ts

```diff
@@ -37,15 +37,14 @@ type Latch = {
 type TestFixtures = {
```

Comment:
Let's land the testing harness changes separately, before this change.

Reply:
done: https://github.com/microsoft/playwright-vscode/pull/514